src/modules/esl-base-element/test/element.test.ts

```diff
@@ -28,17 +29,15 @@ describe('ESLBaseElement', () => {
     }, 0);
   }, 100);
 
-  test('ESLBaseElement register validate', () => {
+  test('ESLBaseElement register validate', async () => {
     // Tag is not empty
     expect(() => TestElement2.register('')).toThrowError();
     TestElement2.register('test-test');
-    return customElements.whenDefined('test-test')
-      .then(() => {
-        expect(() => TestElement2.register('test-test')).not.toThrowError();
-        TestElement2.is = 'test-test-2';
-        // Tag inconsistency
-        expect(() => TestElement2.register('test-test')).toThrowError();
-      });
+    await customElements.whenDefined('test-test');
+    expect(() => TestElement2.register('test-test')).not.toThrowError();
+    expect(() => TestElement2.register('test-test-2')).toThrowError();
+    try {TestElement2.is = 'test-test-2';} catch {}
```

Comment:
shall we add something to ```catch```?

Reply:
I think not.
I'd make an expectation statement here but there is an interesting thing: Node.js throws an error when you try to write immutable property, while browsers just ignore it...